tcs-service/src/main/resources/db/migration/schema/V4.18__adding_postEsrEvent.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE `PostEsrEvent` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `postId` bigint(20) NOT NULL,
+  `eventDateTime` datetime NOT NULL,
+  `filename` varchar(255) NOT NULL,
```

Comment:
This is the field that needs to be nullable.  See discussion about Reconciliation triggered by `tis.post.created` messages.